source/reference/mongosync.txt

```diff
@@ -176,6 +176,73 @@ Global Options
 
    Prints ``mongosync`` version information to stdout.
 
+.. _c2c-cli-OIDC:
+
+Using mongosync with OpenID Connect
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
(information typing)
This should be its own task page. 
It's great to leave the command line options on this reference page, but what you're describing here is a task and we should treat it accordingly. 

As is, this is really hard to find and I'd have a hard time follow what to do.